spec/controllers/posts_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../rails_helper'
```

Comment:
- [ ] [OPTIONAL] Kindly move `posts_spec.rb` and `users_spec.rb` to the `requests` folder instead of the `controllers` folder as this is a recommendation for all tests of type `:request`.